trams_api_spec/trams_api_spec.yaml

```diff
@@ -204,6 +204,24 @@ paths:
             $ref: "#/definitions/AcademyTransferProject"
       security:
       - ApiKeyAuth: []
+  /academyTransferProjects:
+    get:
+      tags:
+        - "Academy Transfer Project"
+      summary: "List academy transfer projects"
+      description: "List truncated versions of academy transfer projects"
+      operationId: "listAcademyTransferProjects"
+      produces:
+        - "application/json"
+      responses:
+        "401":
```

Comment:
Doubt we'd return a 401 from an INDEX style route, it's really 200 or 500 and little else right?

Reply:
I definitely forgot my error codes 😅